pkg/common/cns-lib/vsphere/virtualcentermanager.go

```diff
@@ -145,6 +145,9 @@ func (m *defaultVirtualCenterManager) UnregisterAllVirtualCenters(ctx context.Co
 		if err = m.UnregisterVirtualCenter(ctx, hostInf.(string)); err != nil {
 			log.Warnf("failed to unregister VC %v", hostInf)
 		}
+		if err == ErrVCNotFound {
```

Comment:
I think this should be inside the previous `if` statement

Reply:
done.